packages/cli/src/models/network/Session.ts

```diff
@@ -14,21 +14,33 @@ interface SessionOptions {
   network?: string;
   from?: string;
   timeout?: number;
+  expires?: Date;
 }
 
 const Session = {
 
   getOptions(overrides: SessionOptions = {}): SessionOptions {
     const session = this._parseSession();
-    if (!session) return this._setDefaults(overrides);
+    if (!session || this._hasExpired(session)) return this._setDefaults(overrides);
     log.info(`Using session with ${describe(omitBy(session, (v, key) => overrides[key]))}`);
+
     return { ...session, ...overrides };
   },
 
-  open({ network, from, timeout }: SessionOptions, expires: number = DEFAULT_EXPIRATION_TIMEOUT): void {
+  setDefaultNetworkIfNeeded(network: string): void {
+    const session = this._parseSession();
+    if (!session || this._hasExpired(session)) this.open({ network }, 0, false);
+  },
+
+  getDefaultNetwork(): string | undefined {
+    const session = this._parseSession();
+    return session && this._hasExpired(session) ? session.network : undefined;
```

Comment:
```suggestion
    return session ? session.network : undefined;
```

I think that the session network can still be considered to be the default network regardless of whether it had expired or not.

Reply:
Mmmh but if the session hasn't expired, we won't be prompting the network list (and thus we won't show a default network)

Reply:
Gotcha! I had understood that this was used not for skipping the prompt, but for providing a default option among the networks listed.